builders/integrations/indexers/thegraph.md

```diff
@@ -117,15 +117,15 @@ To deploy your subgraph to Subgraph Studio, change to the subgraph directory in
 3. Authenticate your subgraph with your deploy key. The exact command containing the deploy key can be found on your subgraph's page in Subgraph Studio
 
     ```bash
-    graph auth --studio INSERT_DEPLOY_KEY
+    graph auth INSERT_DEPLOY_KEY
     ```
 
     --8<-- 'code/builders/integrations/indexers/thegraph/terminal/graph-auth.md'
 
 4. Deploy your subgraph and specify the slug for it. Again, you can get the exact command from your subgraph's page in Subgraph Studio
 
     ```bash
-    graph deploy --studio <SUBGRAPH_SLUG>
+    graph deploy <SUBGRAPH_SLUG>
```

Comment:
```suggestion
    graph deploy INSERT_SUBGRAPH_SLUG
```

Not sure why we're not using INSERT_SUBGRAPH_SLUG here, but I think it should be updated